content/english/events/2019-10-17-flatiron.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Techqueria x Flatiron: What does Hispanic Heritage Month really mean?"
+description: "Hispanic Heritage Month ends October 15th. What better way to celebrate and reflect on what it means than to hear from the local community?"
+date: 2019-10-17
+link: https://www.eventbrite.com/e/techqueria-x-flatiron-what-does-hispanic-heritage-month-really-mean-tickets-74494851105?aff=ebdssbdestsearch/
+image: "/assets/img/events/2019-10-17-flatiron.jpg"
+locations:
+  - austin
+organizations:
+  - "Flatiron School"
+---
+
+Come join Techqueria and Flatiron School to reflect on Hispanic Heritage Month!
+We will have a panel discussion on how this celebration of our roots has evolved within our society and how it has been redefined from generation to generation. There will be opportunities to ask questions as well.
+
+There will be food, drinks and mingling so bring a friend and come join us!
+
+Where: 801 Barton Springs Road (Cafe Space that is on ground level)
+Parking: There is a garage attached to the cafe space where the event will be held. It is $5 after 5 pm to enter.
+
+6:00 - 6:30: Set up & panelists arrive
```

Comment:
this would need to be bullet points: https://deploy-preview-557--techqueria.netlify.com/events/2019/10/17/techqueria-x-flatiron-what-does-hispanic-heritage-month-really-mean/

Reply:
not really a blocker right now either though